### PR TITLE
Unclebay contribute

### PR DESCRIPTION
**Pull request by unclebay143**

### CODE REVIEW

File: quiz/quiz-questions.js [hunk 1]
```diff
@@ -125,6 +158,44 @@ const cssQuestions = [
...
+      "If css is a Cascading Stylesheet then which of this will be displayed? h1{color:red} h1{color:blue} h1{color: green}",
+    answers: ["red", "blue", "green"],
+    correctAnswer: "green",
+  },
+  {
+    question:
+      "Which of this color value is not valid?",
+    answers: ["rgba(white)", "rgb(255, 255, 255)", "#fff", "white"],
+    correctAnswer: "rgba(white)",
+  },
```
Comment (sauravk7077) [inline quiz/quiz-questions.js:196]: Remove the ',' after the last '}'

File: quiz/quiz-questions.js [hunk 2]
```diff
@@ -195,7 +266,7 @@ const dataStructure= [
     question: " Pushing an element into stack already having five elements and stack size of 5, then stack becomes?",
     answers: ["Overflow", "Crash","Underflow","User flow"],
     correctAnswer: "Overflow",
-  }
+  },
```
Comment (sauravk7077) [inline quiz/quiz-questions.js:269]: Remove the ',' after the last '}'

**Review by sauravk7077 — CHANGES_REQUESTED** [review]

File: quiz/quiz-questions.js (hunk 1 above)
Comment (unclebay143) [reply quiz/quiz-questions.js]: This has been resolved sir..

**Review by DavidMatthewFraser — APPROVED** [review]
Thanks for the new questions 😄

**Comment by unclebay143** [comment]
> Thanks for the new questions 😄

Happy you loved it ...


Kindly lemme know when new contribution arrive

**Pull request merged**